### ISSUE TITLE
Running this in a server crashes it.

### ISSUE THREAD

**Issue by Ks-Proj** [PssbleTrngle/CreateJetpack#7]
### What happened?

Adding this mod to a, otherwise perfectly fine, server modpack crashes it.

Using this mod in singleplayer has been great, but unfortunately it's not playing nicely in a server setting. Removing this mod allows the server to run perfectly fine. Using the same mods in singleplayer, this mod works perfectly fine as well so I'm not sure it's an incompatibility with another mod.

### Mod Version

create_jetpack-1.1.0

### Minecraft Version

1.18

### Forge Version

40.1.73

### Relevant log output

```shell
A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: main
Stacktrace:
        at com.possible_triangle.create_jetpack.network.ControlManager$Key.<init>(ControlManager.kt:32) ~[create_jetpack-1.1.0.jar%2387!/:1.1.0] {re:classloading,pl:runtimedistcleaner:A}
-- MOD create_jetpack --
Details:
        Caused by 0: java.lang.reflect.InvocationTargetException
                at jdk.internal.reflect.DirectConstructorHandleAccessor.newInstance(DirectConstructorHandleAccessor.java:79) ~[?:?] {}
                at java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499) ~[?:?] {}
                at java.lang.reflect.Constructor.newInstance(Constructor.java:483) ~[?:?] {}
                at net.minecraftforge.fml.javafmlmod.FMLModContainer.constructMod(FMLModContainer.java:67) ~[javafmllanguage-1.18.2-40.1.73.jar%23220!/:?] {}
                at net.minecraftforge.fml.ModContainer.lambda$buildTransitionHandler$4(ModContainer.java:106) ~[fmlcore-1.18.2-40.1.73.jar%23219!/:?] {}
                at java.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java:1804) ~[?:?] {}
                at java.util.concurrent.CompletableFuture$AsyncRun.exec(CompletableFuture.java:1796) ~[?:?] {}
                at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373) ~[?:?] {}
                at java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182) ~[?:?] {}
                at java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655) ~[?:?] {re:computing_frames}
                at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622) ~[?:?] {re:computing_frames}
                at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165) ~[?:?] {}

        Mod File: /D:/Minecraft/3.Servers/Forge/1.18.2/Instances/Kontinuation (beta)/mods/create_jetpack-1.1.0.jar
        Failure message: Create Jetpack (create_jetpack) has failed to load correctly
                java.lang.reflect.InvocationTargetException: null
        Mod Version: 1.1.0
        Mod Issue URL: NOT PROVIDED
        Exception message: java.lang.NoSuchFieldError: binding
Stacktrace:
        at com.possible_triangle.create_jetpack.network.ControlManager$Key.<init>(ControlManager.kt:32) ~[create_jetpack-1.1.0.jar%2387!/:1.1.0] {re:classloading,pl:runtimedistcleaner:A}
        at com.possible_triangle.create_jetpack.network.ControlManager$Key.<init>(ControlManager.kt:22) ~[create_jetpack-1.1.0.jar%2387!/:1.1.0] {re:classloading,pl:runtimedistcleaner:A}
        at com.possible_triangle.create_jetpack.network.ControlManager$Key.<clinit>(ControlManager.kt:23) ~[create_jetpack-1.1.0.jar%2387!/:1.1.0] {re:classloading,pl:runtimedistcleaner:A}
        at com.possible_triangle.create_jetpack.capability.JetpackLogic.<clinit>(JetpackLogic.kt:31) ~[create_jetpack-1.1.0.jar%2387!/:1.1.0] {re:classloading}
        at com.possible_triangle.create_jetpack.Content.register(Content.kt:144) ~[create_jetpack-1.1.0.jar%2387!/:1.1.0] {re:classloading}
        at com.possible_triangle.create_jetpack.CreateJetpackMod.<init>(CreateJetpackMod.java:14) ~[create_jetpack-1.1.0.jar%2387!/:1.1.0] {re:classloading}
        at jdk.internal.reflect.DirectConstructorHandleAccessor.newInstance(DirectConstructorHandleAccessor.java:67) ~[?:?] {}
        at java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499) ~[?:?] {}
        at java.lang.reflect.Constructor.newInstance(Constructor.java:483) ~[?:?] {}
        at net.minecraftforge.fml.javafmlmod.FMLModContainer.constructMod(FMLModContainer.java:67) ~[javafmllanguage-1.18.2-40.1.73.jar%23220!/:?] {}
        at net.minecraftforge.fml.ModContainer.lambda$buildTransitionHandler$4(ModContainer.java:106) ~[fmlcore-1.18.2-40.1.73.jar%23219!/:?] {}
        at java.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java:1804) ~[?:?] {}
        at java.util.concurrent.CompletableFuture$AsyncRun.exec(CompletableFuture.java:1796) ~[?:?] {}
        at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373) ~[?:?] {}
        at java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182) ~[?:?] {}
        at java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655) ~[?:?] {re:computing_frames}
        at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622) ~[?:?] {re:computing_frames}
        at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165) ~[?:?] {}

...

Crash Report UUID: d612e073-9025-4717-8b4b-4217ce74bc8e
        FML: 40.1
        Forge: net.minecraftforge:40.1.73[16:36:20] [main/ERROR] [minecraft/Main]: Failed to start the minecraft server
net.minecraftforge.fml.LoadingFailedException: Loading errors encountered: [
        Create Jetpack (create_jetpack) has failed to load correctly
?7java.lang.reflect.InvocationTargetException: null
]
        at net.minecraftforge.fml.ModLoader.waitForTransition(ModLoader.java:222) ~[fmlcore-1.18.2-40.1.73.jar%23219!/:?] {}
        at net.minecraftforge.fml.ModLoader.lambda$dispatchAndHandleError$21(ModLoader.java:187) ~[fmlcore-1.18.2-40.1.73.jar%23219!/:?] {}
        at java.util.Optional.ifPresent(Optional.java:178) ~[?:?] {re:mixin}
        at net.minecraftforge.fml.ModLoader.dispatchAndHandleError(ModLoader.java:187) ~[fmlcore-1.18.2-40.1.73.jar%23219!/:?] {}
        at net.minecraftforge.fml.ModLoader.lambda$gatherAndInitializeMods$11(ModLoader.java:164) ~[fmlcore-1.18.2-40.1.73.jar%23219!/:?] {}
        at java.lang.Iterable.forEach(Iterable.java:75) ~[?:?] {re:mixin}
        at net.minecraftforge.fml.ModLoader.gatherAndInitializeMods(ModLoader.java:164) ~[fmlcore-1.18.2-40.1.73.jar%23219!/:?] {}
        at net.minecraftforge.server.loading.ServerModLoader.load(ServerModLoader.java:32) ~[forge-1.18.2-40.1.73-universal.jar%23223!/:?] {re:classloading}
        at net.minecraft.server.Main.main(Main.java:112) ~[server-1.18.2-20220404.173914-srg.jar%23218!/:?] {re:classloading,re:mixin,pl:mixin:A}
        at jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104) ~[?:?] {}
        at java.lang.reflect.Method.invoke(Method.java:577) ~[?:?] {}
        at net.minecraftforge.fml.loading.targets.CommonServerLaunchHandler.lambda$launchService$0(CommonServerLaunchHandler.java:32) ~[fmlloader-1.18.2-40.1.73.jar%2317!/:?] {}
        at cpw.mods.modlauncher.LaunchServiceHandlerDecorator.launch(LaunchServiceHandlerDecorator.java:37) [modlauncher-9.1.3.jar%235!/:?] {}
        at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:53) [modlauncher-9.1.3.jar%235!/:?] {}
        at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:71) [modlauncher-9.1.3.jar%235!/:?] {}
        at cpw.mods.modlauncher.Launcher.run(Launcher.java:106) [modlauncher-9.1.3.jar%235!/:?] {}
        at cpw.mods.modlauncher.Launcher.main(Launcher.java:77) [modlauncher-9.1.3.jar%235!/:?] {}
        at cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:26) [modlauncher-9.1.3.jar%235!/:?] {}
        at cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:23) [modlauncher-9.1.3.jar%235!/:?] {}
        at cpw.mods.bootstraplauncher.BootstrapLauncher.main(BootstrapLauncher.java:149) [bootstraplauncher-1.0.0.jar:?] {}
```

**Comment by PssbleTrngle** [comment]
This bug should hopefully already be fixed in Version 1.1.1

**Comment by Ks-Proj** [comment]
> 

Yup, confirmed a server running the mod with the 1.1.1 version doesn't crash on start up. Excited to use this mod now, thanks!
